### PR TITLE
fix some names

### PR DESCRIPTION
**Pull request by kyle-intelletive**
Fix some names to ensure that all of these things are named & cased correctly
- Gatekeeper
- Kubesec
- kube-bench
- kube-hunter

Finishes up ticket 6780

### CODE REVIEW

File: docs/content/en/docs/Reference/dash-parameters.md [hunk 1]
```diff
@@ -105,6 +105,6 @@ For details about configuring environment variables for the dash app, see the [a
 | Variable            | Description           | Default |
 |-------------------|-------------------|------|
 | ADMISSION_CONTROLLER_DEFAULT_ACTION | The default behavior when we fail to validate whether or not an image is compliant for some reason. | deny |
-| GATEKEEPER_TEMPLATE_DIR | Where gatekeeper templates should be loaded from | dist/gatekeeper-templates |
+| GATEKEEPER_TEMPLATE_DIR | Where Gatekeeper templates should be loaded from | dist/gatekeeper-templates |
 | KUBEBENCH_CONFIG_DIR | Where to load the kubebench configuration templates | kube-bench-templates |
```
Comment (becky-intelletive) [inline docs/content/en/docs/Reference/dash-parameters.md:109]: can change to kube-bench here too

File: selenium-testing/test/specs/cluster/kube-bench.spec.ts [hunk 2]
```diff
@@ -82,7 +82,7 @@ describe('KubeBench Page::', () => {
         // @ts-ignore
         await $("//div[contains(@class, 'cdk-overlay-container')]//button/span[contains(text(),'Done')]").customClick('done');
         expect(await $("//div[contains(@class, 'cdk-overlay-container')]//button/span[contains(text(),'Done')]")).not.toBePresent(
-            {message: "Kube Hunter Run configuration window should not be visible"}
+            {message: "kube-hunter Run configuration window should not be visible"}
```
Comment (becky-intelletive) [inline selenium-testing/test/specs/cluster/kube-bench.spec.ts:85]: should this be "kube-bench"?

File: docs/content/en/docs/Reference/dash-parameters.md (hunk 1 above)
Comment (kyle-intelletive) [reply docs/content/en/docs/Reference/dash-parameters.md]: Nice catch, fixed in latest commit

File: selenium-testing/test/specs/cluster/kube-bench.spec.ts (hunk 2 above)
Comment (kyle-intelletive) [reply selenium-testing/test/specs/cluster/kube-bench.spec.ts]: fixed in latest commit

**Review by becky-intelletive — APPROVED** [review]

**Pull request merged**